src/styles/index.css

```diff
@@ -0,0 +1,8 @@
+@import url(./product-card.css);
+@import url(./stars.css);
+@import url(./reset.css);
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
```

Comment:
better set typography in `html{}`, so if you set `font-size` in `html{}` the value will become a `1rem` equivalent 😉